### PR TITLE
Add support for docker Compose V2

### LINKED ISSUE #986: "make init" doesn't work
Has the command ever worked without adding `-f compose.yml -f compose.override.yml` to the `docker-compose` commands?

```
$ make init
ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
        
make: *** [Makefile:8: init] Fehler 1
```

### PR DESCRIPTION
**Pull request by piotrkardasz**
To change or to use the fallback version add to terminal configuration:
```sh
export DOCKER_COMPOSE="docker-compose"
```

resolve #986

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,40 +1,46 @@
...
+	@ENV=$(ENV) DOCKER_USER=$(DOCKER_USER) $(DOCKER_COMPOSE) run --rm -i nodejs sh
 
 node-watch:
-	@ENV=$(ENV) DOCKER_USER=$(DOCKER_USER) docker-compose run --rm -i nodejs "npm run watch"
+	@ENV=$(ENV) DOCKER_USER=$(DOCKER_USER) $(DOCKER_COMPOSE) run --rm -i nodejs "npm run watch"
+
+docker-compose-check:
+	@which $(DOCKER_COMPOSE) > /dev/null || (echo "Please install docker compose binary" && exit 1)
+	@echo "You are using \"$(DOCKER_COMPOSE)\" binary"
+	@echo "Current version is \"$$($(DOCKER_COMPOSE) version)\""
```
Comment (Prometee) [inline Makefile:47]: Do you think it can be used at the very begening of the `init` ?

Comment (piotrkardasz) [reply Makefile:47]: Sure, I will add it

File: Makefile [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
-	@ENV=$(ENV) DOCKER_USER=$(DOCKER_USER) docker-compose run --rm nodejs
-	make install
-	@ENV=$(ENV) DOCKER_USER=$(DOCKER_USER) docker-compose up -d
+	@ENV=$(ENV) DOCKER_USER=$(DOCKER_USER) $(DOCKER_COMPOSE) run --rm php composer install --no-interaction --no-scripts
+	@ENV=$(ENV) DOCKER_USER=$(DOCKER_USER) $(DOCKER_COMPOSE) run --rm nodejs
+	@make -s install
+	@ENV=$(ENV) DOCKER_USER=$(DOCKER_USER) $(DOCKER_COMPOSE) up -d
 
 run:
 	make up
```
Comment (Prometee) [inline Makefile]:
```suggestion
	@make -s up
```

I could be wrong but can we kept the same syntax you already used also here ?

Comment (piotrkardasz) [reply Makefile]: Sure ✔️

**Review by Wojdylak — APPROVED** [review]

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "add-support-for-docker-compose-v2"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by Prometee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Piotr! :tada:

**Pull request merged**